db/hints/sync_point.cc

```diff
@@ -135,12 +136,12 @@ sstring sync_point::encode() const {
     ser::serializer<sync_point_v1>::write(measure, v1);
 
     // Reserve version_size bytes for the version
-    bytes serialized{bytes::initialized_later{}, version_size + measure.size()};
-    seastar::simple_memory_output_stream out{reinterpret_cast<char*>(serialized.data()), version_size + measure.size()};
+    sstring serialized{sstring::initialized_later{}, version_size + measure.size()};
+    seastar::simple_memory_output_stream out{serialized.data(), version_size + measure.size()};
     ser::serializer<uint8_t>::write(out, 1);
     ser::serializer<sync_point_v1>::write(out, v1);
 
-    return base64_encode(serialized);
+    return base64_encode(reinterpret_cast<int8_t*>(serialized.data(), serialized.size()));
```

Comment:
```
reinterpret_cast<int8_t*>(serialized.data(), serialized.size())
```

Is valid C++? `reinterpret_cast` takes only one parameter. Please make sure that all of the commits compile.

Reply:
Done.